api-authorizations/src/main/scala/com/normation/plugins/apiauthorizations/ApiAccountsExtension.scala

```diff
@@ -131,10 +152,18 @@ class ApiAccountsExtension(val status: PluginStatus)(implicit val ttag: ClassTag
               appAcl.ports.giveAcl.subscribe(function(acl) {
                 app.ports.getCheckedAcl.send(acl)
               });
+            }
+            app.ports.initAcl.subscribe(async function () {
+              await initAcl({id: "account.id", acl: []});
             });
             //get the acl list of the selected account
-            app.ports.shareAcl.subscribe(function(acl){
-              appAcl.ports.getToken.send(acl)
+            app.ports.shareAcl.subscribe(async function(token){
+              // only initialize with a new token when modal is opened with a known token id :
+              // this is an implicit contract between app and appAcl
+              if (appAcl !== undefined && token.id !== "") {
+                await initAcl(token);
+                appAcl.ports.getToken.send(token)
+              }
```

Comment:
the check for `token.id !== ""` was necessary, because it seems that the `Accounts` app on Rudder make the plugin load an empty token every time (so when an existing one with custom ACLs, it appears with empty acl)